components/contact/ContactForm.vue

```diff
@@ -271,7 +292,7 @@
               v-model="contactInfo.orgAddress2"
               class="formInput"
               type="text"
-              placeholder="Adress"
+              placeholder="Address"
```

Comment:
Not ready for translation.

Reply:
Ignore this. I'm going to log it as an issue. The whole form is using had coded translations for placeholders.